deploy/kubernetes/metricbeat-kubernetes.yaml

```diff
@@ -258,6 +293,8 @@ metadata:
   labels:
     k8s-app: metricbeat
 spec:
+  # Set to 0 if using leader election provider with the Daemonset
```

Comment:
What happens if we remove the Deployment instead? I don't think the UX is great here...

Reply:
🤔 Removing the deployment completely will have the same result -> 0 Deployment Pods.